setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import sys
+import os
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from disutils.core import setup
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, "README.md")) as f:
+    README = f.read()
+
+
+setup(name="smatch",
+      version="1.0",
+      description="Smatch (semantic match) tool",
+      long_description=README,
+      author="Shu Cai",
+      author_email="shucai@isi.edu",
```

Comment:
Can you use author_email as "shucai.work@gmail.com"?

Reply:
Of course. Done. When I'm adding you on PyPI, though, it says `User 'shucai' does not have a verified primary email adddress and cannot be added as a Owner for project.`. I think you need to verify your email address.

Reply:
Yes, thanks for the reminder. I've verified the email address.